pkgs/applications/virtualization/conmon/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, glib
+, glibc
+, systemd
+}:
+
+stdenv.mkDerivation rec {
+  project = "conmon";
+  name = "${project}-${version}";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = project;
+    rev = "v${version}";
+    sha256 = "08fgkbv7hq62lcq39za9pm2s2j92ismgrkvfm7acwbvajqh9syjb";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ glib systemd ] ++
+    stdenv.lib.optionals stdenv.hostPlatform.isMusl [ glibc glibc.static ];
```

Comment:
I'm confused here. Shouldn't this be `(!stdenv.hostPlatform.isMusl)` instead?

Reply:
Absolutely, thanks for the hint. I fixed it.